src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string = 'triangle';
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
No need in this. Fix everywhere

Reply:
I see but it won't let me write without blanks. Am I missing something?
<img width="634" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/143909365/1f3e53a7-d8ee-4120-9e0d-d938853c1155">

Reply:
I mean you can completely get rid of it